server/services/GithubWorkflowService.js

```diff
@@ -174,7 +194,7 @@ const createGithubWorkflow = async ({ job, directory, gitSha, atVersion }) => {
       'X-GitHub-Api-Version': '2022-11-28'
     },
     data: JSON.stringify({
-      ref: 'main',
+      ref: 'self-hosted',
```

Comment:
Do you want to revert this before pushing to staging @ChrisC?

Reply:
We can keep this on `staging` but should def revert in a separate PR before pushing to prod!

Reply:
Cool submitted #1457 to track that